Path-Check/gaen-mobile
# Update exposure detail styles

#### Description:
Fixes a few visual regressions on the exposure detail screen
- Adjust spacing to be more even
- Fix the svg icon to be more adjustable and design flexible
- Changes main container to be scrollview to account for larger text accessibility



#### Linked issues:
https://trello.com/c/fJOPzesP



#### Screenshots:


![Simulator Screen Shot - iPhone 8 - 2020-07-30 at 13 47 32](https://user-images.githubusercontent.com/32882075/88956428-535a9380-d26b-11ea-8019-198a3341177d.png)

#### How to test:
- Navigate to exposure detail screen
- Note that the screen is now a scrollview
- Change text size to large
- Note the screen is still functional at larger text sizes
